src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -28,6 +38,15 @@ const InputText = ({
   return (
     <div className={'form-group ' + className}>
       <label htmlFor={inputId}>{label}</label>
+      {helperText !== '' && (
+        <p className="form-group__p--margin-zero">
+          {helperText}
+          <a href={linkHref} rel="noreferrer external" target="_blank">
+            {linkText}
+            {icon === true && <FontAwesomeIcon className="helper-text__link-icon" icon={faUpRightFromSquare} />}
```

Comment:
`icon` as a named prop sounds like if the component is accepting an icon object. Whereas your usage seems weather or not using an icon. I would advice renaming the prop to `showTargetIcon` or `hasIcon` types